README.md

```diff
@@ -29,14 +38,18 @@ The example uses the [World Cities Dataset](https://simplemaps.com/data/world-ci
   }
 ```
 
-### Database Setup
+## Database setup
 
 It's recommended to use [`Postgres`] for your database so the schema can use `BigInt` and `Float` nicely.
 
 Also, the seed script uses Prisma's [`createMany()`](https://www.prisma.io/docs/reference/api-reference/prisma-client-reference#createmany) to bulk load records in batches which is significantly faster that loading data row by row -- which is important because we have ~43,000 world cities in the dataset.
 
 Unfortunately, `createMany` is not supported by SQLite.
 
+### Setting up a database locally
+
```

Comment:
[question] Are there any "gotchas" in using a local Postgres database that we might want to surface here? (I have no idea, ha)

Reply:
I'm not sure as I haven't dabbled much in databases, but @dthyresson might be aware of some as it pertains to developing with the Redwood framework

Reply:
Chatted with David, no known problems so we're all good here 🚀